integrationTests/vm/txsFee/multiShard/asyncESDT_test.go

```diff
@@ -102,10 +102,10 @@ func TestAsyncESDTTransferWithSCCallShouldWork(t *testing.T) {
 	_, err = testContextSender.Accounts.Commit()
 	require.Nil(t, err)
 
-	expectedAccumulatedFees = big.NewInt(189880)
+	expectedAccumulatedFees = big.NewInt(189890)
```

Comment:
here also. What changed in terms of fees?
If these changes are required because the activation flag is set for the transferAndAsyncCallbackFixEnableEpoch, then, I think we should double the tests, one with the active flag and one with the disabled flag in order to easily assure backwards compatibility.

Reply:
because the consumption was wrong. it was completely wrong for different reasons - like 2 fixes in process.go